content/en/docs/contribute/analytics.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Viewing site analytics
```

Comment:
Hi @a-mccarthy .
Can you update the title field to match the site's style, "Viewing Site Analytics". Thanks.

Reply:
addressed, thanks!